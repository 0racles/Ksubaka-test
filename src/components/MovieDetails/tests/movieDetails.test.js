import { MovieDetails } from '../index';

const props = {
  items: [],
};

describe('Movie Details', () => {
  it('should match snapshot', () => {
    const renderedItem = shallow(<MovieDetails {...props} />);
    expect(renderedItem).toMatchSnapshot();
  });
});
