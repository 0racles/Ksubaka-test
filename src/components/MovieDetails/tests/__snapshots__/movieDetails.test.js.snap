// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Details should match snapshot 1`] = `
<.sc-bdVaJa
  className="movieDetails-ul"
/>
`;
